.travis.yml

```
@@ -16,7 +16,7 @@ script:
   - docker run --rm -it -v $PWD/example_project:/src -w /src -e TOXENV -e TOX_PARAMS="-p auto" fedorapython/fedora-python-tox
   # Test that we can install (multiple) packages from DNF as devel dependencies. Without those, cffi and pyaml are not installable from sources.
   # This test overrides the entrypoint, so it has to be specified manually as the first command. It always runs system pip.
-  - 'docker run --rm -it -e DNF_INSTALL="libffi-devel libyaml-devel" fedorapython/fedora-python-tox sh -c "/run_tests.sh; pip install -I --no-deps --compile --no-binary :all: cffi pyyaml"'
+  - 'docker run --rm -it -e DNF_INSTALL="libffi-devel pkgconfig(libffi) /usr/bin/cowsay" fedorapython/fedora-python-tox sh -c "/run_tests.sh; pip install -I --no-deps --compile --no-binary :all: cffi pyyaml; cowsay DONE"'
```

This should make sure the pip command actually works:

```suggestion
  - 'docker run --rm -it -e DNF_INSTALL="libffi-devel pkgconfig(libffi) /usr/bin/cowsay" fedorapython/fedora-python-tox sh -c "/run_tests.sh; pip install -I --no-deps --compile --no-binary :all: cffi pyyaml && cowsay DONE"'
```